code/game/machinery/doors/door.dm

```diff
@@ -262,11 +262,6 @@
 	operating = 0
 	air_update_turf(1)
 	update_freelok_sight()
-
-	//I shall not add a check every x ticks if a door has closed over some fire.
-	var/obj/effect/hotspot/fire = locate() in loc

```

Comment:
Eh, I forget things, I think it did that at one point- either way it doesn't matter to remove this or keep it, but my point is I did leave it there on purpose.